src/main/resources/mockdata/README

```diff
@@ -1 +1,15 @@
+00ed10af-fac2-46ad-9f94-a298358646a2.json - A simple placeholder record.
```

Comment:
I think you missed the point...  The idea is that since we chose to use the PO UUID for the json file names, they don't really provide any context.  To help clarify what each json file represents, we list the files in this README and add a short description/explanation.  The values in the third row ("Purpose") of the excel spreadsheet should suffice.... e.g. "One-time ebook" or "Ongoing Print Media" etc.